src/vs/workbench/contrib/notebook/browser/view/renderers/commonViewComponents.ts

```diff
@@ -3,12 +3,13 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-import { renderCodicons } from 'vs/base/common/codicons';
+import * as DOM from 'vs/base/browser/dom';
 import { MenuEntryActionViewItem } from 'vs/platform/actions/browser/menuEntryActionViewItem';
 import { MenuItemAction } from 'vs/platform/actions/common/actions';
 import { IContextMenuService } from 'vs/platform/contextview/browser/contextView';
 import { IKeybindingService } from 'vs/platform/keybinding/common/keybinding';
 import { INotificationService } from 'vs/platform/notification/common/notification';
+import { renderCodicons } from 'vs/base/common/codicons';
```

Comment:
`browser` - there is potential for slips given those names are the same... Tho, I have no better suggestion.

Reply:
`renderCodiconsAsElement`, `renderCodiconsHTMLElement`?

Reply:
I like `renderCodiconsAsElement`